bot/cogs/coding/reference.py

```diff
@@ -87,7 +92,7 @@ async def _git_main_ref(part: str, ctx: Context, text: str) -> Message:
     base_url = f"https://git-scm.com/docs/{part}{text}"
     url = urllib.parse.quote_plus(base_url, safe=";/?:@&=$,><-[]")
 
-    async with aiohttp.ClientSession() as client_session:
+    async with aiohttp.ClientSession as client_session:
```

Comment:
why did u remove the parentheses

Reply:
sorry, will fix 😅

Reply:
I'm marking this resolved as I've referenced this same line in my requested change where I suggesting replacing this whole thing with `bot.aio_session` which seems more reasonable if, for some reason that will not be done, this should be re-opened